package.json

```diff
@@ -33,7 +33,7 @@
     "@web-std/fetch": "^4.1.0",
     "@web3-storage/data-segment": "^5.1.0",
     "@web3-storage/filecoin-client": "^3.3.3",
-    "@web3-storage/w3up-client": "^12.5.3",
+    "@web3-storage/w3up-client": "^14.0.0-rc.0",
```

Comment:
this will break integration tests, maybe you can skip them as I am working on this